stubs/croniter/METADATA.toml

```diff
@@ -1,2 +1,2 @@
-version = "3.0.3"  # keep micro version pinned, changes often
+version = "~=3.0.4"
```

Comment:
This is a bit doubtful, as there still seem to be compatible changes between each release, so it may be safer to use an exact match. But technically the changes are forward compatible.

Reply:
IIRC we've had the daily test broken several times by croniter; I'd honestly just prefer to keep it pinned to a micro version